lib/stellar_core_commander/process.rb

```diff
@@ -413,6 +438,33 @@ def dump_metrics
       dump_server_query("metrics")
     end
 
+    Contract String, Symbol, Num, Num, Or[Symbol, Num] => Any
+    def record_scp_data(fname, txtype, accounts, txs, txrate)
+      m = metrics
+      fname = "#{working_dir}/#{fname}"
+
+      run_data = [txtype, accounts, txs, transactions_applied, txrate]
+      run_data.push(m["loadgen.step.submit"]["mean_rate"])
+      run_data.push(m["loadgen.step.submit"]["mean"])
+      run_data.push(m["scp.timing.nominated"]["mean"])
+      run_data.push(m["scp.timing.nominated"]["max"])
+      run_data.push(m["scp.timing.externalized"]["mean"])
+      run_data.push(m["scp.timing.externalized"]["max"])
+      run_data.push(m["ledger.ledger.close"]["mean_rate"])
+      run_data.push(m["ledger.ledger.close"]["mean"])
+      run_data.push(max = m["ledger.ledger.close"]["max"])
+
+      write_csv fname, METRICS_HEADER unless File.file?(fname)
```

Comment:
can you move the definition of METRICS_HEADER closer to this call site so that it's easy to see that the columns match?